.github/workflows/release.yml

```diff
@@ -53,6 +53,65 @@ jobs:
     outputs:
       ENSO_RELEASE_ID: ${{ steps.prepare.outputs.ENSO_RELEASE_ID }}
       ENSO_VERSION: ${{ steps.prepare.outputs.ENSO_VERSION }}
+  enso-build-ci-gen-job-deploy-runtime-linux-x86_64:
+    name: Upload Runtime to ECR (linux, x86_64)
```

Comment:
maybe `Upload Runtime to Cloud AWS ECR (linux, x86_64)`

Reply:
Unless you find the name misleading, I'd rather not make it any longer. It is already the longest name in the workflow and horizontal space for job names is rather limited. Making it longer will just cause GitHub to trim it when displaying in the navigation bar.
In the job graph it is already trimmed to 
![obraz](https://github.com/enso-org/enso/assets/1548407/adb32735-d42f-4220-8dc6-3c8f7c4abb64)